barretenberg/ts/src/barretenberg/backend.ts

```diff
@@ -286,35 +286,19 @@ export class UltraHonkBackend {
     return await this.api.acirHonkSolidityVerifier(this.acirUncompressedBytecode, new RawBuffer(vkBuf));
   }
 
-  // TODO(https://github.com/noir-lang/noir/issues/5661): Update this to handle Honk recursive aggregation in the browser once it is ready in the backend itself
   async generateRecursiveProofArtifacts(
-    // eslint-disable-next-line @typescript-eslint/no-unused-vars
-    _proof: Uint8Array,
-    // eslint-disable-next-line @typescript-eslint/no-unused-vars
-    _numOfPublicInputs: number,
-  ): Promise<{ proofAsFields: string[]; vkAsFields: string[]; vkHash: string }> {
+    proof: Uint8Array,
+    publicInputs: string[],
```

Comment:
If pub inputs is just passed along from input to output is there any value in having it here?